src/app/ApplicationTemplate.Shared.Views/Framework/DeviceInformationProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Windows.System.Profile;
+
+namespace ApplicationTemplate;
+
+public sealed class DeviceInformationProvider : IDeviceInformationProvider
```

Comment:
Duplicate file?

Reply:
Hum not sure, let me check.

Reply:
Indeed it was a duplicate. Don't know why it was there.